tests/test_config_section_virt.py

```diff
@@ -112,14 +112,22 @@ def test_validate_unencrypted_password(self):
         result = self.virt_config._validate_unencrypted_password('password')
         self.assertIsNone(result)
 
-    def test_validate_unicode_unencrypted_password(self):
+    @patch('virtwho.config.requests')
+    def test_validate_unicode_unencrypted_password(self, _mock_requests):
         """
         Test of validation of password that is not encrypted and it contains some
         UTF-8 string.
+        Passes or fails based on python-requests version
         """
         self.init_virt_config_section()
         self.virt_config['password'] = 'Příšerně žluťoučký kůň pěl úděsné ódy.'
+        # Verision that can't handle unicode
+        _mock_requests.__version__ = '2.19.0'
         result = self.virt_config._validate_unencrypted_password('password')
+        self.assertIsNotNone(result)
```

Comment:
What should the result be if not None?
I would suggest being as specific as we can to ensure proper behaviour.